src/middlewared/middlewared/api/v25_04_0/static_route.py

```diff
@@ -0,0 +1,54 @@
+from middlewared.api.base import BaseModel, Excluded, excluded_field, ForUpdateMetaclass
+
+from pydantic import IPvAnyAddress, IPvAnyNetwork
+
+
+__all__ = [
+    "StaticRouteEntry",
+    "StaticRouteUpdateArgs",
+    "StaticRouteUpdateResult",
+    "StaticRouteCreateArgs",
+    "StaticRouteCreateResult",
+    "StaticRouteDeleteArgs",
+    "StaticRouteDeleteResult",
+]
+
+
+class StaticRouteEntry(BaseModel):
+    destination: IPvAnyNetwork
+    gateway: IPvAnyAddress
+    description: str = ""
+    id: int
+
+
+class StaticRouteCreate(StaticRouteEntry):
+    id: Excluded = excluded_field()
+
+
+class StaticRouteCreateArgs(BaseModel):
+    data: StaticRouteCreate
+
+
+class StaticRouteCreateResult(BaseModel):
+    result: StaticRouteEntry
+
+
+class StaticRouteUpdate(StaticRouteEntry, metaclass=ForUpdateMetaclass):
```

Comment:
It's usually a good practice to inherit update schema from create schema. That's how we used to do this in the previous API and it worked well. The benefit in this specific case: no need to exclude `id` twice

Reply:
Yeah, I find it weird to inherit from the create class. I fixed it though. ps. this is why I only like 1 positional argument for our public methods 😄 (instead of passing `id` as the 1st positional arg. if we didn't do that it would reduce the boilerplate for the new api paradigm)